Utilities/rXml.cpp

```diff
@@ -18,37 +18,50 @@ rXmlNode::rXmlNode(void *ptr)
 	handle = ptr;
 }
 
+// These functions go unused.
+/*
 rXmlNode::rXmlNode(const rXmlNode& other)
 {
 	ownPtr = true;
-	handle = reinterpret_cast<void *>(new wxXmlNode(*reinterpret_cast<wxXmlNode*>(other.handle)));
+	handle = reinterpret_cast<void *>(new pugi::xml_node(*reinterpret_cast<pugi::xml_node*>(other.handle)));
 }
 
 rXmlNode &rXmlNode::operator=(const rXmlNode& other)
 {
 	if (ownPtr)
 	{
-		delete reinterpret_cast<wxXmlNode*>(handle);
+		delete reinterpret_cast<pugi::xml_node*>(handle);
 	}
-	handle = reinterpret_cast<void *>(new wxXmlNode(*reinterpret_cast<wxXmlNode*>(other.handle)));
+	handle = reinterpret_cast<void *>(new pugi::xml_node(*reinterpret_cast<pugi::xml_node*>(other.handle)));
 	ownPtr = true;
 	return *this;
 }
+*/
 
 rXmlNode::~rXmlNode()
 {
 	if (ownPtr)
 	{
-		delete reinterpret_cast<wxXmlNode*>(handle);
+		delete reinterpret_cast<pugi::xml_node*>(handle);
```

Comment:
The types for handle are now specified, and all instances of reinterpret_cast have been removed. It seems as though there is no situation where handle _needs_ to be a void*, so I'll code in the accessor, but I don't think we need it.